src/components/Accordion/Accordion.tsx

```diff
@@ -16,76 +17,81 @@ type Props = {
    */
   className?: string;
   /**
-   * Additional classnames passed in for styling active only when open.
+   * Outline variant adds extra padding and a rounded border.
    */
-  classNameOpen?: string;
+  hasOutline?: boolean;
   /**
-   * Additional classnames passed in for styling active only when not open.
+   * Used to specify which heading element should be rendered for each Accordion.Title child.
    */
-  classNameClosed?: string;
-  /**
-   * Whether panel is expanded by default.
-   */
-  defaultOpen?: boolean;
+  headingAs: HeadingElement;
   /**
    * Compact variant shrinks the Accordion size.
    */
   variant?: 'compact';
```

Comment:
Btw if there's only one variant, does it make more sense to rename the prop to be specific to that variant? Ie `isCompact?: boolean`  vs `variant?: 'compact'`. Just putting the question out there; I'm not asking you to change it.

Reply:
@dierat This is a good callout! We actually have two variants with one hidden. `compact` and non-`compact` which is the default if `variant` is missing. [Our guidelines](https://github.com/chanzuckerberg/edu-design-system/blob/main/docs/CODE_GUIDELINES.md#variants) don't make it super clear what to do in the case of default variants... seems to imply "when variant is missing, apply default style"?

In this case we may end up with a bigger variant at some point in the future, so I think having this prop allows for that without as much refactoring work. We might also consider applying a default/regular class when no specific variant is applied, which we might be doing elsewhere.

Reply:
>We actually have two variants with one hidden.

Sorry, that's what I meant—one default and one alternate variant, which could easily be a boolean.

> In this case we may end up with a bigger variant at some point in the future

Then maybe we should call the "compact" size "sm" and the default "md" so it's easier to tack on a "lg" later on?

Reply:
I'm down to call the prop `size` and the options `sm` and `md`, aligns with the mocks better too